### DISCUSSION TITLE
支持多种导出格式

### DISCUSSION THREAD

**Discussion by nieqika** [Hufe921/canvas-editor#750] [Ideas]
考虑增加多种导出格式，如导入了word 文本编辑后，再出为word格式。

**Comment by Hufe921** [comment]
后续扩展插件

**Reply by jordanparker6** [reply]
@Hufe921 been exploring this repo. The docx import is the current barrier to adoption. Too many styles are lost. If you have capacity, we could sponsor some work on this feature? Contact me a` jordan.parker@lawme.ai` if interested. I have opened a ticket here and have started exploring some work on it.

https://github.com/Hufe921/canvas-editor-plugin/issues/20

**Reply by Wuper310** [reply]
> 后续扩展插件

开发者你好，能否考虑合作？  我们可以支付一定的报酬，期待答复谢谢。 qq2296856
